packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -1548,6 +1552,38 @@ class FocusManager with DiagnosticableTreeMixin, ChangeNotifier {
   /// [FocusManager] notifies.
   void removeHighlightModeListener(ValueChanged<FocusHighlightMode> listener) => _highlightManager.removeListener(listener);
 
+  /// Adds a key event handler to a set of handlers that are called before any
+  /// key event handlers in the focus tree are called.
+  ///
+  /// All of the handlers in the set will be called for every key event the
+  /// [FocusManager] receives. If any one of the handlers returns
+  /// [KeyEventResult.handled] or [KeyEventResult.skipRemainingHandlers], then
+  /// none of the handlers in the focus tree will be called.
```

Comment:
Maybe link to removeEarlyKeyEventHandler, e.g. in a see also section?

Reply:
Maybe also link to addLateKeyEventHandler (and vice versa) just for discoverability?

Reply:
For completeness: can you also document the behavior if handlers are added/removed from a OnKeyEventCallback (i.e. while the callbacks are being invoked)?